leptonai/ui/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/lep-docs/leptonai.svg" />
```

Comment:
we have one in `assets/logo.svg`. maybe consolidate these two?

Reply:
`leptonai.svg` is a svg consists of a logo as well as an inline text, so it doesn't seem to be able to replace the existing `logo.svg`